gpt_2_simple/gpt_2.py

```diff
@@ -294,15 +293,13 @@ def generate_samples():
                 feed_dict={context: batch_size * [context_tokens]})
             for i in range(min(sample_num - index, batch_size)):
                 text = enc.decode(out[i])
-                text = '======== SAMPLE {} ========\n{}\n'.format(
-                    index + 1, text)
+                text = f'======== SAMPLE {index + 1} ========\n{text}\n'
                 all_text.append(text)
                 index += 1
         print(text)
         maketree(os.path.join(SAMPLE_DIR, run_name))
         with open(
-                os.path.join(SAMPLE_DIR, run_name,
-                             'samples-{}').format(counter), 'w') as fp:
+                os.path.join(SAMPLE_DIR, run_name, f"samples-{counter}.txt", 'w') as fp:
```

Comment:
Once again, more readable in the code, slightly smaller filesize, and added .`txt` file extension for accessibility in Colaboratory.